lib/arm/crc32_impl.h

```diff
@@ -151,12 +152,66 @@ crc32_pmull_aligned(u32 remainder, const uint8x16_t *p, size_t nr_segs)
 #include "../crc32_vec_template.h"
 #endif /* PMULL implementation */
 
+#undef DISPATCH_INSTR
+#if (defined(__ARM_FEATURE_CRC32) ||	\
+    (ARM_CPU_FEATURES_ENABLED &&	\
+     COMPILER_SUPPORTS_CRC32_TARGET_INTRINSICS)) && \
+     /* not yet tested on big endian, probably needs changes to work there */ \
+    (defined(__BYTE_ORDER__) && __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__)
```

Comment:
How about just putting in the `le64_bswap()`s that are needed for it to work on big endian?